library/common/src/main/java/com/epmedu/animeal/common/di/AnalyticsModule.kt

```diff
@@ -0,0 +1,27 @@
+package com.epmedu.animeal.common.di
+
+import android.content.Context
+import com.epmedu.animeal.common.analytics.AnalyticsManager
+import com.epmedu.animeal.common.analytics.AnalyticsManagerImpl
+import com.google.firebase.analytics.FirebaseAnalytics
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
```

Comment:
let's create separate library module `analytics`

Reply:
Yeah, I thought about it.
But from my point of view Analytics is common part of all screens and we will not win a lot from splitting it on separate modules. Isn't it?

Reply:
Resolved by reason: We try to remove all interactions with `common` module